eng/common/scripts/SemVer.ps1

```diff
@@ -138,6 +168,7 @@ class AzureEngSemanticVersion {
     # Python uses no separators and "b" for beta so this sets up the the object to work with those conventions
     $this.PrereleaseLabelSeparator = $this.PrereleaseNumberSeparator = ""
```

Comment:
Should we also set the BuildNumberSeparator here and in the other convention?

Reply:
The regex should handle this, but this is more defensive so I'll add it.